### PR TITLE
session.c: updated dbus activation environment with more env vars

### PR DESCRIPTION
**Pull request by winerysearch**
This PR resolves issue #694.
(This is my very first open source contribution.)

### CODE REVIEW

**Comment by Consolatis** [comment]
Thanks!
The first string needs to have a space at the end otherwise it will show up as `.. XDG_CURRENT_DESKTOPXCURSOR_SIZE ..`.

This PR may need to wait a bit until after the release (targeted for March 1st).

**Comment by johanmalm** [comment]
@winerysearch Good effort and thanks for contributing.

If you can a change (like @Consolatis suggested) just run

```
git add src/session.c
git commit --amend
git push -f origin dbus_act_env
```

...and they'll make it to this PR.

**Comment by johanmalm** [comment]
Also, if you run `./scripts/check --file=src/session.c` you can run the code style checks locally.

Currently we get `src/config/session.c:128: WARNING: break quoted strings at a space character`

**Comment by winerysearch** [comment]
Thanks both of you @Consolatis @johanmalm

**Comment by johanmalm** [comment]
Looks good. If you're okay to wait for a week or so, we'll just sort out the release first. Always best to make changes like this at the start of the merge window.

**Comment by winerysearch** [comment]
Sure no problem

**Comment by ahesford** [comment]
Handling of session environment variables changed with the merge of #1543. Please update this PR to simply add the new variables to the [`env_vars` list at the top of `config/session.c`](https://github.com/labwc/labwc/blob/master/src/config/session.c#L19-L24) and this should be good to go.

**Comment by winerysearch** [comment]
I updated the env_vars list appropriately.

File: src/config/session.c [hunk 1]
```diff
@@ -20,7 +20,10 @@ static const char *const env_vars[] = {
 	"DISPLAY",
 	"WAYLAND_DISPLAY",
 	"XDG_CURRENT_DESKTOP",
-	NULL,
+	"XCURSOR_SIZE",
+	"XCURSOR_THEME",
+	"XDG_SESSION_TYPE",
+	"LABWC_PID"
```
Comment (ahesford) [inline src/config/session.c:26]: This list needs a NULL terminator to signal its end:
```suggestion
	"LABWC_PID",
	NULL
```

**Comment by winerysearch** [comment]
You're right. I corrected it.

**Comment by Consolatis** [comment]
Thanks.

Great job for your first open source contribution :)

**Pull request merged**